Dockerfile.development

```diff
@@ -41,5 +44,5 @@ EXPOSE 5000
 # tini is the init process that will adopt orphaned zombie processes
 # e.g. chromium when launched to create a new PDF
 ENTRYPOINT [ "tini", "--" ]
-# Create local S3 buckets before building the app
-CMD npm run docker-dev
+# Create local AWS resources before building the app
+CMD sh init-localstack.sh && npm run docker-dev
```

Comment:
just to double check, this would work for `docker-compose up` too?

Reply:
yup, but you'll need to `npm run dev` the very first time for `docker-compose` to build the latest version of the `Dockerfile`